src/components/TimerButton/WorkBreakButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import socket from "../socket";
+
+const WorkBreakButton = ({ roomName }: { roomName: string }): JSX.Element => {
+	// It may be needed to fetch from the server whether the user is on a break or not.
+	// The default isBreak state below is just a placeholder for now.
+	const [isBreak, setBreak] = useState(false);
+
+	const onClickHandler = (): void => {
+		// const userName = localStorage.getItem('');
+		if (!isBreak) {
+			socket.emit("breakTimer", {
+				userName: "name-functionality-is-yet-to-be-implemented",
+				roomName,
+			});
+			setBreak(true);
```

Comment:
```suggestion
```

We dont want to setBreak here. We would want to update setBreak when the server responds with the isBreak message.

This is because we want the source of truth to be the server, so that all clients are consistent. If done like above, some clients might be inconsistently set as break

see: https://github.com/CommunityFocus/CommunityFocus/issues/84

It might be harder for you to test the functionality of the button you just created, but thats fine. From what i see, it seems like it would work fine for the server to respond and then change the state.

Reply:
I should leave out the setBreak altogether since this is going to be addressed in another issue?